test/compflags/checks/checkModByZero.chpl

```diff
@@ -13,47 +13,47 @@ record R {
 }
 
 /* Define this so that %= is automatically defined */
-proc %(lhs: R, rhs: R) return new R(lhs.x % rhs.x);
+operator R.%(lhs: R, rhs: R) return new R(lhs.x % rhs.x);
 
 var rec1 = new R(1),
     rec0 = new R(0);
 
 select testParamCases {
-  //proc %(param a: int(?w), param b: int(w)) param
+  //operator R.%(param a: int(?w), param b: int(w)) param
```

Comment:
Hi, a `sed` probably replaced most of these to be `operator R.%`...

They should probably be just `operator %`